frontend/src/lib/k8s/apiProxy.ts

```diff
@@ -1334,7 +1351,7 @@ interface StreamParams {
   additionalProtocols?: string[];
 }
 
-function connectStreamWithParams(
+async function connectStreamWithParams(
```

Comment:
I see this was changed to async, but I don't see any call sites changed. Can you please confirm if the call sites are also changed? (do they now need await?)

It would be good if you can document at least the changes to this function.

Reply:
Added documentation. I checked, the calls do not need await, they are working as expected

Reply:
Thanks, fixed it